ambuda/utils/user_mixins.py

```diff
@@ -59,3 +63,7 @@ def is_moderator(self) -> bool:
     @property
     def is_admin(self) -> bool:
         return self.has_role(SiteRole.ADMIN)
+
+    @property
+    def is_active(self) -> bool:
```

Comment:
I like the idea of this property. For the name, `active` doesn't feel quite right, as that could be confused with user activity in general. Maybe something like `is_ok`